.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
+# Mark the database schema as having been generated.
+db/schema.rb linguist-generated
```

Comment:
There is an error in the migration. Kindly see the error below which stopped the project from running. Please clone the project and try to run it locally so that you can get the error fixed.
![db](https://user-images.githubusercontent.com/35504149/153271863-d077b92c-03bc-4f04-9d78-52b2ba70a0b1.png)